### DISCUSSION TITLE
只需要接收邮件，怎么配置

### DISCUSSION THREAD

**Discussion by soeos** [dreamhunter2333/cloudflare_temp_email#424] [General]
## 然后访问网页上就可以看到邮件，不需要添加用户等，如何配置

**Comment by dreamhunter2333** [comment]
登录admin 后台即可查看所有邮件

**Comment by dreamhunter2333** [comment]
<img width="967" alt="image" src="https://github.com/user-attachments/assets/51425c11-822a-4c48-aa00-99f373702cf5">
